src/js/components/DataTable/TableCell.jsx

```diff
@@ -41,20 +43,22 @@ const TableCell = ({
 
   if (tooltip) {
     cellValue = (
-      <div className="d-flex">
+      <div className={`d-flex ${customTooltipStyles}`}>
```

Comment:
`customTooltipStyles` -> `tooltipClassname`, styles mean that it should be passed to inline styles.